src/app/models/theming.model.ts

```diff
@@ -0,0 +1,19 @@
+export interface ApplicationTheme {
+    appTitle: { name: string; url: string; };
+    backgroundColor: string;
+    logo: 'judicial' | 'myhmcts' | 'default' | 'none';
```

Comment:
Can we extract this into an enum

Reply:
Thank you @kuda1992, I have now implemented the changes as you suggested.